backend/server.js

```diff
@@ -0,0 +1,53 @@
+const app = require("./app");
+const mongoose = require("mongoose");
+
```

Comment:
The server.js file has now been moved into app.js and server.js. This means that we can import the routing logic from app.js, as seen above. It also means that we can instantiated the server using environment variables as seen below.